### PR TITLE
Jormungandr: add items_per_schedule param to /stop_schedules

### PR DESCRIPTION
**Pull request by antoine-de**
This param sets a maximum number of date_times per route/point in /stop_schedules

linked to http://jira.canaltp.fr/browse/NAVP-490

---

This change is [Reviewable](https://reviewable.io/reviews/canaltp/navitia/1480)

### CODE REVIEW

**Comment by pbougue** [comment]
Jenkins, test this please.

File: source/time_tables/tests/departure_boards_test.cpp [hunk 1]
```diff
@@ -458,7 +459,7 @@ BOOST_FIXTURE_TEST_CASE(test_not_matched_cal, small_cal_fixture) {
 
     navitia::PbCreator pb_creator(*(b.data), bt::second_clock::universal_time(), null_time_period);
     departure_board(pb_creator, "stop_point.uri=stop1", std::string("empty_cal"), {}, d("20120615T080000"),
-                    86400, 0, std::numeric_limits<int>::max(), 1, 10, 0, nt::RTLevel::Base);
+                    86400, 0, std::numeric_limits<int>::max(), 10, 0, nt::RTLevel::Base, std::numeric_limits<size_t>::max());
```
Comment (pbougue) [inline source/time_tables/tests/departure_boards_test.cpp]: minor (+l482, 513, 551): line length

**Comment by TeXitoi** [comment]
OK for me.

**Comment by antoine-de** [comment]
Jenkins, test this please

**Pull request merged**